docs/custom-images-contract.rst

```diff
@@ -4,28 +4,57 @@
 Custom Images contract
 ======================
 
-A contract is a set of requirements that any crawler custom Docker image have to comply with to be able to run on Scrapy Cloud.
+A contract is a set of requirements that any crawler custom `Docker`_ image have to comply
+with to be able to run on Scrapy Cloud.
 
-Scrapy crawler Docker images are already supported via the :ref:`scrapinghub-entrypoint-scrapy <sh-entrypoint-scrapy>` contract implementation. If you want to run crawlers built using other framework/language than Scrapy/Python, you have to make sure your image follows the contract statements listed below.
+Scrapy crawler Docker images are already supported via
+the :ref:`scrapinghub-entrypoint-scrapy <sh-entrypoint-scrapy>` contract implementation.
+If you want to run crawlers built using other framework/language than Scrapy/Python,
+you have to make sure your image follows the contract statements listed below.
 
+.. _Docker: https://docs.docker.com/
 
 Contract statements
 -------------------
 
 1. Docker image should be able to run via ``start-crawl`` command without arguments.
+   ``start-crawl`` should be :ref:`executable and located on the search path <scripts-example>`.
 
 .. code-block:: bash
 
     docker run myscrapyimage start-crawl
 
 2. Docker image should be able to return a spiders list via ``list-spiders`` command without arguments.
+   ``list-spiders`` should be :ref:`executable and located on the search path <scripts-example>`.
 
 .. code-block:: bash
 
    docker run myscrapyimage list-spiders
 
 3. Crawler should be able to get all needed params using :ref:`system environment variables <environment-variables>`.
 
+.. _scripts-example:
+
+.. note::
+
+    The simplest way to place scripts on the search path is to create a
+    symbolic link to the script located in the directory present in the `PATH`_
+    environment variable. Here's an example `Dockerfile`_:
+
+    .. code-block:: dockerfile
+
+       FROM python:3
+       RUN mkdir -p /spiders
+       WORKDIR /spiders
+       ADD . /spiders
+       # Create a symbolic link in /usr/sbin because it's present in the PATH
+       RUN ln -s /spiders/start-crawl /usr/sbin/start-crawl
+       RUN ln -s /spiders/list-spiders /usr/sbin/list-spiders
+       # Make scripts executable. Assuming that start-crawl and list-spider
```

Comment:
Thanks, I started writing it before I decided to provide an example, just forgot to remove.